low-level/files/opt/seagate/sspl/setup/setup_rabbitmq_cluster.py

```diff
@@ -15,21 +15,28 @@
 # about this software or licensing, please email opensource@seagate.com or
 # cortx-questions@seagate.com.
 
+###############################################
+# Joins nodes in current node Rabbitmq cluster
+###############################################
```

Comment:
We can remove these lines. If needed, please put inside class doc.

Reply:
This was the ask.

Reply:
yes, needed.